.coveragerc

```diff
@@ -0,0 +1,10 @@
+[run]
+branch = True
+parallel = True
+omit =
+	conf\.py
+
+[report]
+omit =
+	conf\.py
```

Comment:
what is `conf\.py`?

Reply:
in the `doc` directory.

Reply:
But yeh  dunno if this will reach it or not, or if we need to bother including this. Looking.

Reply:
Actually I can't remember where I got the idea to include this.

Reply:
I think it is so `conf.py` & `conftest.py` are not covered but shouldn't that need `conf*.py`?

Reply:
I don't think the doc directory is covered at all, do we need to bother with this?

Reply:
if you want it to match also `conftest.py` you do need the wildcard. also there is no need to escape the `.`, the syntax being used here is glob's not regular expressions. I don't think you need to omit this file anyway

Reply:
> I don't think the doc directory is covered at all, do we need to bother with this?

the `doc/` directory doesn't get installed in site-packages so it is not visible to pytest nor coverage

Reply:
Removed all omit entries for this PR. I am hoping we can merge this today!

Reply:
waiting for CI to finish to have a last quick look but I think this PR is ready to be merged